README.md

```diff
@@ -13,16 +14,24 @@ Google Docs.
 Compatibility
 -------------
 
-This library should work on any modern browser that supports `Blobs` (either natively or via 
+This library should work on any modern browser that supports `Blobs` (either natively or via
 [Blob.js](https://github.com/eligrey/Blob.js/)). It was tested on Google Chrome 36, Safari 7 and
 Internet Explorer 10.
 
 It also works on Node.js (tested on v0.10.12) using `Buffer` instead of `Blob`.
 
+Images Support
+-------------
+
+This library supports only inlined base64 images (sourced via DATA URI). But it is easy to covert a
+regular image (source from static folder) on the fly. If you need example of coverting images sourced
```

Comment:
I think we can drop "(source...)" here. You are writing about this in the next sentence